test/conftest.py

```diff
@@ -0,0 +1,44 @@
+import pytest
+import time
+from common import start_logging_plugin, \
+    kill_logging_plugin
+
+
+def pytest_addoption(parser):
+    parser.addoption("--splunkd-url",
+                     help="splunkd url used to send test data to. \
+                          Eg: https://localhost:8089",
+                     default="https://52.53.254.149:8089")
```

Comment:
Can we please remove the real IP and update the default address with sudo value? 
May be leave default as localhost.